charts/incubator/ariang/values.yaml

```diff
@@ -0,0 +1,15 @@
+image:
+  repository: leonismoe/ariang
+  pullPolicy: IfNotPresent
+  tag: 1.2.4
+
+securityContext:
+  readOnlyRootFilesystem: false
+
+service:
+  main:
+    protocol: HTTP
+    ports:
+      main:
```

Comment:
Protocol is a port thing, not service.
There are apps that is wrong currently tho, and on my todo to fix it.